docs/modules/ROOT/pages/explanation.adoc

```diff
@@ -0,0 +1,84 @@
+= Explanations
+
+== What does the operator do exactly?
+
+The main interaction point with the operator is through the API. The operators job is to keep inventory about all the clusters and tenants that are managed using SYN. It will also bootstrap the Vault secrets and Git repositories necessary for SYN to work correctly.
```

Comment:
```suggestion
The main interaction point with the operator is through the API. The operator's job is to keep inventory about all the clusters and tenants that are managed using SYN. It will also bootstrap the Vault secrets and Git repositories necessary for SYN to work correctly.
```